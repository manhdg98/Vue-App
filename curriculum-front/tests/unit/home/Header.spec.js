import { mount, RouterLinkStub } from '@vue/test-utils'
import Header from '@/components/home/Header.vue'

describe('home/Header.vue', () => {
  test('if snapshot is correct', () => {
    const wrapper = mount(Header, {
      stubs: {
        RouterLink: RouterLinkStub
      }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
