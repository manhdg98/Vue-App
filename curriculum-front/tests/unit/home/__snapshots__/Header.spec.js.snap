// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home/Header.vue if snapshot is correct 1`] = `
<div class="overflow-hidden v-sheet theme--light" style="height: 400px; position: relative;">
  <div class="container fill-height">
    <div class="row no-gutters align-center justify-center">
      <div class="text-center col">
        <h1 class="display-3 mb-6">Curricula App</h1>
        <h3 class="display-1 mb-6">
          Create, share, and track your learning curricula.<br>
          All for free!
        </h3> <a class="v-btn v-btn--depressed v-btn--flat v-btn--outlined v-btn--router theme--light v-size--large primary--text"><span class="v-btn__content">
          Sign Up
        </span></a>
      </div>
    </div>
  </div>
</div>
`;
